### ISSUE TITLE
Temperature polling crashes on 14.0.0+

### ISSUE THREAD

**Issue by p-sam** [retronx-team/sys-clk#53]
```
[2022-03-23 17:59:20.276] [!] ASSERT_RESULT_OK: tsGetTemperatureMilliC
  RC: [0xf601] 0001-0123
  in /home/zachary/sys-clk/sysmodule/src/clocks.cpp:325
```

**Comment by p-sam** [comment]
Should be fixed in the next sys-clk release, although all temperature but "Skin" will no longer have decimal precision

**Comment by p-sam** [comment]
Fixed by commit b4482ccc14ce446a089754af1a9d731642b885b6
